front/src/applications/osrd/views/OSRDConfig/TimetableSelector.js

```diff
@@ -15,19 +15,22 @@ import { updateTimetableID } from 'reducers/osrdconf';
 const timetableURL = '/timetable/';
 
 export default function TimetableSelector(props) {
-  const { mustUpdateTimetable } = props;
-  const [selectedTimetable, setselectedTimetable] = useState(undefined);
+  const { mustUpdateTimetable, setMustUpdateTimetable } = props;
+  const [selectedTimetable, setSelectedTimetable] = useState(undefined);
+  const [isWorking, setIsWorking] = useState(false);
   const [trainList, setTrainList] = useState(undefined);
   const { timetableID } = useSelector((state) => state.osrdconf);
```

Comment:
more specific selector please, comp is hydrated on every change on osrdconf

Reply:
state.osrdconf.timetableID at least

Reply:
Resolved with https://github.com/DGEXSolutions/osrd/pull/2586/commits/c567887d4bdfff8fdb176425c355a7857ee84aa9